bld/namelist_files/namelist_defaults_pop.xml

```diff
@@ -1438,6 +1441,9 @@
 <nox_flux_monthly_input%filename ocn_grid="gx1v7" ocn_bgc_config="cesm2.0">ocn/pop/gx1v6/forcing/ndep_ocn_1850_w_nhx_emis_gx1v6_c180427.nc</nox_flux_monthly_input%filename>
 <nhy_flux_monthly_input%filename ocn_grid="gx1v7" ocn_bgc_config="cesm2.0">ocn/pop/gx1v6/forcing/ndep_ocn_1850_w_nhx_emis_gx1v6_c180427.nc</nhy_flux_monthly_input%filename>
 
+<nox_flux_monthly_input%filename ocn_grid="tx0.1v3">ocn/pop/tx0.1v3/tmp/ndep_ocn_1850_w_nhx_emis_tx0.1v3_c191115.nc</nox_flux_monthly_input%filename>
+<nhy_flux_monthly_input%filename ocn_grid="tx0.1v3">ocn/pop/tx0.1v3/tmp/ndep_ocn_1850_w_nhx_emis_tx0.1v3_c191115.nc</nhy_flux_monthly_input%filename>
```

Comment:
Need to either move this to proper directory (and import it to inputdata repo) or generate updated version